modules/checklist/assets/js/app/components/checklist-wrapper.js

```diff
@@ -2,27 +2,33 @@ import { useState } from 'react';
 import { List } from '@elementor/ui';
 import CheckListItem from './checklist-item';
 import PropTypes from 'prop-types';
+import SuccessMessage from './success-message';
+import { isStepChecked } from '../../utils/functions';
 
 const ChecklistWrapper = ( { steps, setSteps } ) => {
 	const [ expandedIndex, setExpandedIndex ] = useState( -1 );
+	const checkIfAllStepsCompleted = steps.filter( isStepChecked ).length === steps.length;
```

Comment:
`Check if` implies a function
```suggestion
	const isChecklistCompleted = steps.filter( isStepChecked ).length === steps.length;
```

Reply:
done